service/interface.go

```diff
@@ -10,7 +10,7 @@ import (
 type Manager interface {
 	CreateUser(usr *domain.User) *domain.Error
 	UpdateUser(usr *domain.User) error
-	ListUser(limit int32, name string) []*domain.User
+	ListUser(int32, map[string]interface{}) []*domain.User
```

Comment:
keep the parameters name 
it will increase the readability

Reply:
Done. Kindly Check